ocs_ci/helpers/helpers.py

```diff
@@ -270,6 +272,8 @@ def create_pod(
             only if dc_deployment is True
         subpath (str): Value of subPath parameter in pod yaml
         deployment (bool): True for Deployment creation, False otherwise
+        scc (dict): Security context for fsGroup, runAsUer, runAsGroup
```

Comment:
There is another parameter "security_context" in this function which is set for the container. Please mention that "scc" is for the pod . OR please update the description of "security_context" to mention that it is sent on container.